plugins/modules/event_stream.py

```diff
@@ -51,6 +51,19 @@
     type: str
     aliases:
       - type
+  headers:
+    description:
+      - Comma separated HTTP header keys that you want to include in the event payload.
```

Comment:
Does it make sense to have a dictionary instead of a comma-separated string? Not a blocker, just curious to know.

Reply:
A dictionary would be cleaner to write, internally we can always transform it into a comma-separated string. Not a blocker.

Reply:
what should be the key and values of that content? It would make more sense a list instead.

Reply:
something similar to - 

```
    headers:
      Content-Type: application/json
      x-ms-client-workspace-path: /subscriptions/{{ sub_id }}/resourcegroups/{{ res_group }}/providers/microsoft.operationalinsights/workspaces/{{ w_spaces }}
      x-ms-client-platform: ibiza
      x-ms-client-auth-token: "{{ token_az }}"
```

Reply:
I think you may be misunderstanding the purpose of that parameter. It is a list of headers that come in the POST request made to the Event Stream endpoint that are included in the event for ansible-rulebook. 
A dict doesn't make sense because the value comes from the request. It should be a list of headers anyway, which is what the API expects.

Reply:
OK. Make sense.